x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -24,28 +24,31 @@ const StyledTransactionLink = styled(TransactionDetailLink)`
 `;
 
 interface Props {
-  items: ITransactionGroup[];
+  items: TransactionGroup[];
   isLoading: boolean;
 }
 
-const traceListColumns: Array<ITableColumn<ITransactionGroup>> = [
+const traceListColumns: Array<ITableColumn<TransactionGroup>> = [
   {
     field: 'name',
     name: i18n.translate('xpack.apm.tracesTable.nameColumnLabel', {
       defaultMessage: 'Name',
     }),
     width: '40%',
     sortable: true,
-    render: (name: string, { sample }: ITransactionGroup) => (
-      <EuiToolTip id="trace-transaction-link-tooltip" content={name}>
+    render: (_: string, { sample }: TransactionGroup) => (
+      <EuiToolTip
+        id="trace-transaction-link-tooltip"
```

Comment:
I know this line was just name changes, but still: I don't think the id `trace-transaction-link-tooltip` is necessary anymore. I don't see it referenced anywhere, and i don't think it's used in any snapshot testing.